BrainPortal/script/show_cbrain_rev

```diff
@@ -34,25 +34,10 @@ end
 
 # Code lifted almost as-is from cbrain_file_revision.rb
 
-    git_tag = nil
-    seen    = {}
     Dir.chdir(top) do
-      tags_set = `git tag -l`.split.shuffle # initial list: all tags we can find
-      git_tag = tags_set.shift unless tags_set.empty? # extract one as a starting point
-      seen[git_tag]=true
-      while tags_set.size > 0
-        tags_set = `git tag --contains '#{git_tag}'`.split.shuffle.reject { |v| seen[v] }
-        git_tag = tags_set.shift unless tags_set.empty? # new first
-        seen[git_tag]=true
-      end
-      if git_tag
-        num_new_commits = `git rev-list '#{git_tag}..HEAD'`.split.size
-      elsif default_tag # fallback for plugins, when -z used
-        num_new_commits = `git rev-list HEAD`.split.size
-        git_tag = default_tag
-      end
-      if git_tag
-        git_tag += "-#{num_new_commits}" if num_new_commits > 0
+      out = `git describe --tags --long`
+      if out
+        git_tag, _, _ = out.rpartition('-')
```

Comment:
why do you use this strange statement when in the other code you used `out.rpartition('-')[0]` ?

Reply:
to be clean, redirect stderr in the git command

Reply:
ok for '-'. that is a common pattern with python, so I used with it.

Reply:
sorry not follow sderr to std out ? or null?

Reply:
To /dev/null . Redirect the command's stderr to /dev/null so we don't pollute the output with spurious error messages. **Just like everywhere else in this file!**

Reply:
adding command to that file (not sure for another)